### ISSUE TITLE
Use fortran-lang test-drive

### ISSUE THREAD

**Issue by ivan-pi** [grimme-lab/nlopt-f#21]
https://github.com/grimme-lab/nlopt-f/blob/8e36c15a6d83e11d615702ae5865c7ebe447e4de/fpm.toml#L14

**Comment by awvwgk** [comment]
Yeah, we should update this. At least my fork stays up-to-date with upstream automatically, so there is no real loss.
